### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Rabbitdmr** [rustdesk/rustdesk#5493] [General]
Greetings can you be so kind to show me the way to get started

**Comment by dinger1986** [comment]
Read the docs https://rustdesk.com/docs/en/

**Reply by Rabbitdmr** [reply]
Can someone show me the way

**Comment by d-damien** [comment]
The website has become a bit confusing recently, idk why. The software itself is really simple : it works like Teamviewer. The receiver starts Rustdeskt & gives their Rustdesk number to the helper, who enters it in the field. After a few seconds, either a password prompt appears and the same happens for the password, or the receiver clicks on "Accept connection".

That's it, the helper has control on the receiver PC.

**Reply by dinger1986** [reply]
whats confusing? Any suggestions what can be changed?

**Reply by d-damien** [reply]
I understand the team might be working hard, and it is easy to have confirmation bias on one's own work, but the main audience of this − end users − is easily confused by the new site. In particular, _it all assumes everybody knows Teamviewer_.

The download page shows only small links (tiny in comparison to that huge anti-scam banner). The doc says starts immediately about "opening command line" and "self-hosting" (do you know how scary that sounds to end users ?). The video tutorials section is similar.

I would suggest to make very few assumptions as the average end user barely knows their OS name. Maybe a basic video tutorial on the front page ?

But the old site wasn't bad I think. What was the motivation for its dismissal ?

**Reply by dinger1986** [reply]
ok fair enough, can look at that and see if we can make it more accessible for users.

**Reply by d-damien** [reply]
Thank you for your consideration.

**Reply by d-damien** [reply]
IMHO the old download page is the most important.

**Reply by dinger1986** [reply]
this is a juggling act between scammers using the software, the more usable it is for users the more usable it is for scammers!

**Reply by d-damien** [reply]
I hear you, a banner is necessary. But I'm thinking about the OS logos.

**Reply by dinger1986** [reply]
yeah we looked at that, even had a detection so it would install the right version but opted against it due to scammers, we dont want to make it easy for them

**Comment by d-damien** [comment]
The video tutorials section has an example (buried in some later chapter... and ignore the custom server part) :

https://youtu.be/9nzHm3xGz2I?si=9dNKoOxpY8doN7RS&t=1372
